ru-intl/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+The RuIntl Russian international is a set of Cyrillic and Latin layouts unified in composition and arrangement of the punctuation marks and special characters.
+
+The Cyrillic layout allows you to type text in the following languages:
+- Russian;
+- Belarusian;
+- Bulgarian;
+- Serbian;
+- Ukrainian.
+
+Languages ​​available on the Latin layout:
+- English;
+- Hungarian;
+- Dutch;
+- Danish;
+- Spanish;
+- Italian;
+- Lithuanian;
+- Maltese;
+- German;
+- Polish;
+- Portuguese;
+- Romanian;
+- Slovak;
+- Slovenian;
+- Turkish;
+- Finnish;
+- French;
+- Czech;
+- Swedish;
+- Estonian.
+
+# Details
```

Comment:
Just to be clear - are these standard layouts in common use?

Reply:
Yes. In terms of popularity those layouts are still considered as the exotic ones, but they have potential because at the moment it's the only multilingual keyboard layouts set where the Russian keyboard based Cyrillic layout completely matches (the punctuation marks and the special characters location is unified for both of those layouts) with the QWERTY based latin one. Those layouts were just merged into the XKB repository and will be available for the Linux users in the XKB version 2.42. I hope that one day the XKB version 2.42 will be integrated in the Chrome OS to make those layouts native for the Chrome OS. Before those layouts become native for the Chrome OS I would like to make them available for the Chrome OS users via the Chrome browser extensions

Reply:
If all you want is to make it available, there is no need to add the code here - you can just create and publish your own Chrome extension.

Reply:
That's true, but I would like to contribute to the community. Before creating those extensions I searched an information everywhere in internet and fortunately I found this code repository. Using the extensions here which were developed by another contributors I was able to create my own extensions to start using those layouts and I also would like to become a contributor. I think that the extensions I created which work well for me in my day-to-day life will be useful for the community. First of all, I propose to use the AltGr in the Compose key mode. There is no interference with the left Alt key (standard one) or with any Ctrl keys that allows to use the ChromeOS key shortcuts as before. One more functionality I added is the capsoffkeys array is used to store the non-letter key (for example the top row digit keys) codes to avoid having the upper registry characters even if the caps lock is activated. The ruintl_ru layout is using the shit and the AltGr key modifiers. The ruintl_en layout is more complex because it contains  the dead keys. It will allow to newcomers to get a good template to start working on their own extensions. What about the code modifications I made to meet your requirements? Are they correct? Should I make some more modifications?